### PR TITLE
feat: neovim compatablity

### PR DESCRIPTION
**Pull request by isabelroses**
:)

### CODE REVIEW

**Comment by comfysage** [comment]
wait.... you wrote docs?

**Comment by comfysage** [comment]
bell: *does not write docs ever*
*gets called out for her behavior*
bell: :angry:

its okay, babe
you can just not write docs; ill do it for you <3

File: README.md [hunk 1]
```diff
@@ -13,6 +13,44 @@
 2. Run `hx --grammar fetch` to fetch the grammar from this repository, and then `hx --grammar build` to build the grammars.
 3. Run `just helix sync` or `just helix link` to copy or symlink the [queries](./queries) to Helix's runtime directory, as unfortunately [queries are not used from the grammar repository by default](https://github.com/helix-editor/helix/discussions/11379#discussioncomment-10194806).
 
+### neovim
```
Comment (uncenter) [inline README.md]:
```suggestion
### Neovim
```

File: README.md [hunk 2]
```diff
@@ -13,6 +13,44 @@
...
+
+#### Lazy.nvim installation
```
Comment (uncenter) [inline README.md]:
```suggestion
#### Lazy.nvim
```

File: README.md [hunk 3]
```diff
@@ -13,6 +13,44 @@
...
+
+Extend your treesitter plugin with the tera plugin.
```
Comment (uncenter) [inline README.md]:
```suggestion
Extend your nvim-treesitter plugin with this Tera plugin.
```

File: README.md [hunk 4]
```diff
@@ -13,6 +13,44 @@
...
+    dependencies = {
+        ...
+        { "uncenter/tree-sitter-tera", build = ":TSUpdate tera" },
+        ...
+    },
+    build = ":TSUpdate",
+},
+```
+
+#### Manual installation
```
Comment (uncenter) [inline README.md]:
```suggestion
#### Manual
```

File: README.md [hunk 5]
```diff
@@ -13,6 +13,44 @@
...
+  install_info = {
+    url = "https://github.com/uncenter/tree-sitter-tera",
+    files = { "src/parser.c" },
+    branch = "main",
+  },
+  filetype = "tera",
+}
+```
+
+Run `:TSInstall tera` in neovim to install the above parser.
```
Comment (uncenter) [inline README.md]:
```suggestion
Run `:TSInstall tera` in Neovim to install the above parser.
```

**Review by uncenter — CHANGES_REQUESTED** [review]

**Review by uncenter — APPROVED** [review]

**Pull request merged**